src/components/sky-header/sky-header.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'sky-header',
+  templateUrl: 'sky-header.html',
+})
+export class SkyHeaderComponent implements OnInit {
+  @Input() title: string;
+  @Input() hours: number;
+  @Input() money: number;
+  @Input() secMoney: number;
+  @Input() sky: number;
+
+  ngOnInit() {
+    this.hours = this.hours || 0;
```

Comment:
We don't want to default to 0, a value of null is easier to handle (means value has not (yet) been retrieved)